qbox.sql

```diff
@@ -188,6 +188,7 @@ CREATE TABLE IF NOT EXISTS `player_contacts` (
 CREATE TABLE IF NOT EXISTS `players` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `citizenid` varchar(50) NOT NULL,
+  `cid` int(11) DEFAULT NULL,
```

Comment:
We shouldn't actually put this back right as it will break existing users of Qbox? Instead as long as we have the PlayerData.cid present it will be backwards compatible

Reply:
It will not break as it's an extra field and doesn't remove the cid inside the charinfo and besides that, it's default null so it won't throw errors for the database

Reply:
I've added this back as to have full backwards compatibility as we saw someone having issues with codesign multichar I believe using the database instead of the core

Reply:
ok